faqs/isnt-pos-less-secure-or-more-centralized-than-pow.md

```diff
@@ -3,6 +3,24 @@ title: Isn't proof-of-stake (PoS) less secure/decentralized than proof-of-work (
 weight: 3.1
 attribution:
   -
-    name:
-    link:
+    name: Yule Andrade
+    link: https://twitter.com/yulesa
 ---
+
+No, PoS can be much more secure/decentralized than PoW.
+
+In PoW, if miners act maliciously, they lose the block reward and the operational cost of running the rigs like electricity bills and capital cost. The blockchain is safer the more the malicious player has to lose. That way rig operational cost and the block reward has to be higher as possible in order to the network to be safer. That’s the reason PoW is so wasteful in electricity resources. Also, the network security is drastically reduced if the block rewards is suddenly decreased.
+
+PoS operational cost is very low. Instead, if you act maliciously, aside of losing the rewards (a positive reinforcement), your stake is also burned (a negative reinforcement).
+
+This has three advantages:
+
+First, it better aligns incentives to the participants in the network considering that the stake is now denominated in the network currency.
+
+Second, since you lose your stake acting badly, if you try it be dishonest again, your next attack is less effective because your stake is now lower and so is your voting power.
```

Comment:
My understanding is that nodes required 32 ETH to participate. Less and you're out - more doesn't matter. There's not impact on voting power by having additional quantities of ETH in your validator node.

Reply:
You can validate with less than 32 ETH for been inactive, for example: https://beaconcha.in/validator/a3729585f2a8e765bc0697490aa21c4e5a021c071e43b38e90a041b1370523e710364eb1cb5fa95ffee3c1ecbc26eb70

But my answer also applies to validators that got slashed. Imagine that a group of validator tried an attack and didn't succeed. They got slashed and kicked out, but if they ever came back after waiting their time-out, now they have less to stake.

Vitalik writes about that in https://vitalik.ca/general/2020/11/06/pos2020.html in sub tittle "Attacks are much easier to recover from in proof of stake".

Maybe I haven't be clear about it on my text.